README.md

```diff
@@ -4,7 +4,8 @@
 
 [![HTML Manuscript](https://img.shields.io/badge/manuscript-HTML-blue.svg)](https://AlexsLemonade.github.io/OpenPBTA-manuscript/)
 [![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf)
-[![Build Status](https://travis-ci.com/AlexsLemonade/OpenPBTA-manuscript.svg?branch=master)](https://travis-ci.com/AlexsLemonade/OpenPBTA-manuscript)
+[![GitHub Actions Status](https://github.com/AlexsLemonade/OpenPBTA-manuscript/workflows/Manubot/badge.svg)](https://github.com/AlexsLemonade/OpenPBTA-manuscript/actions)
+[![Travis Build Status](https://travis-ci.com/AlexsLemonade/OpenPBTA-manuscript.svg?branch=master)](https://travis-ci.com/AlexsLemonade/OpenPBTA-manuscript)
```

Comment:
So I know Travis is currently disabled, will it continue to be disabled and if so, will this status always indicate that the build is unknown?

Reply:
Hmm - probably. Let's just get rid of this :)